gov2/s3/GeneratePresignedURL/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "Bucket": "mygroovybucket",
```

Comment:
This is not an approved example bucket name.

Reply:
Replaced with "aws-docs-example-bucket" and "aws-docs-example-key".